### ISSUE TITLE
Fix: app crash if fails to read any mod file

### ISSUE THREAD

**Issue by AlyxMoon** [AlyxMoon/lunar-factorio-mod-manager#144]
From a message on the Factorio forum post:
(https://forums.factorio.com/viewtopic.php?p=491002#p491002)
```
Hi!)
I've got a crash...

error-log – https://www.dropbox.com/s/x2qwhuu6xkfv2 ... g.txt?dl=0
info-log – https://www.dropbox.com/s/ckjrjraxiuqqu ... g.txt?dl=0

I'd suppose the reason may be my special path names (_STEAM_, _MODS_, _Game Utils_).
```

**Comment by AlyxMoon** [comment]
After troubleshooting I found it was just an error with a corrupted zip file. However, failing to read a mod shouldn't be a complete failure state. The app should be able to continue in the event of a bad mod file.

**Comment by AlyxMoon** [comment]
Corrected this behavior with release 2.3.0
